### PR TITLE
diff: refine log && minor fix

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
### What problem does this PR solve? 

1. fix issue https://github.com/pingcap/tidb-tools/issues/357, don't print this log
2. refine log for check report, the old log like:

`[2020/06/24 16:36:23.332 +08:00] [INFO] [main.go:87] ["check report"] [report="\ncheck result: fail!\n0 tables' check passed, 1 tables' check failed.\n\ntable: diff_test.t_10\ntable's struct equal\ntable's data not equal\n\n"]`

after this PR, it will be:

`[2020/06/24 17:50:47.149 +08:00] [INFO] [report.go:80] ["check result summary"] ["check passed num"=0] ["check failed num"=1]
[2020/06/24 17:50:47.149 +08:00] [INFO] [report.go:87] ["table check result"] [schema=diff_test] [table=t_10] ["struct equal"=true] ["data equal"=false]`

3. a minor fix, after this pr https://github.com/pingcap/tidb-tools/pull/346/files#r444783910, if meet error will continue and doesn't save the check result. 

### Check List 

Tests 

 - Manual test (check log)

### CODE REVIEW

File: sync_diff_inspector/main.go [hunk 1]
```diff
@@ -63,7 +63,7 @@ func main() {
 
 	ctx := context.Background()
 	if !checkSyncState(ctx, cfg) {
-		log.Fatal("sourceDB don't equal targetDB")
+		os.Exit(1)
```
Comment (csuzhangxc) [inline sync_diff_inspector/main.go:67]: Should we need to log something here?

File: sync_diff_inspector/report.go [hunk 2]
```diff
@@ -73,36 +76,19 @@ func (r *Report) String() (report string) {
...
-			if !result.StructEqual {
-				structResult = "table's struct not equal"
-			} else {
-				structResult = "table's struct equal"
-			}
-
-			if !result.DataEqual {
-				dataResult = "table's data not equal"
+			if result.MeetError != nil {
+				log.Info("table check result", zap.String("schema", schema), zap.String("table", table), zap.String("meet error", result.MeetError.Error()))
```
Comment (csuzhangxc) [inline sync_diff_inspector/report.go]: How about using `log.Error`?

File: sync_diff_inspector/main.go (hunk 1 above)
Comment (WangXiangUSTC) [reply sync_diff_inspector/main.go:67]: add in 532137f

File: sync_diff_inspector/report.go (hunk 2 above)
Comment (WangXiangUSTC) [reply sync_diff_inspector/report.go]: update in 532137f

**Review by csuzhangxc — COMMENTED** [review]
LGTM

**Review by july2993 — APPROVED** [review]

**Pull request merged**